apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,159 @@
+    const numberFormat = new Intl.NumberFormat(locale, options)
+    const parts = numberFormat.formatToParts(amount)
+    return parts.map((part) => part.value).join('')
+}
+
+
+const generateTicketDocumentOfPaidWorks = async ({ task, baseAttrs, context, locale, ticket, organization }) => {
+    const dadata = new FinanceInfoClient()
+    let psrn = null, organizationAddress = null, iec = null
+    try {
+        ({ iec, psrn, organizationAddress } = await dadata.getOrganization(organization.tin))
+    } catch (error) {
+        console.error(error)
+    }
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone).locale(locale)
+
+    const contact = ticket.contact
+        ? await getById('Contact', ticket.contact)
+        : null
+    const employee = organization.id && ticket.executor
+        ? await getByCondition('OrganizationEmployee', {
+            organization: { id: organization.id },
+            user: { id: ticket.executor },
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+        status_not: INVOICE_STATUS_CANCELED,
+    }, {
+        sortBy: ['createdAt_ASC'],
+    })
+
+    const currencyCode = get(invoices, '0.currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
+    let totalSum = 0, totalVAT = 0, vat = get(invoices, '0.rows.0.vatPercent')
+
+    const listOfWorks = invoices.reduce((acc, invoice) => {
+        const rows = Array.isArray(invoice.rows) ? invoice.rows : []
+        acc.push(...rows.map((row, index) => {
+            const price = parseFloat(!row.isMin ? row.toPay : null)
+            const sum = price * row.count
+            totalSum += sum
+
+            return {
+                number: index + 1 || '',
+                name: row.name || '',
+                count: String(row.count) || '',
+                price: !Number.isNaN(price) ? renderMoney(price, currencyCode, locale) : '',
+                vat: vat || '',
+                sum: !Number.isNaN(sum) ? renderMoney(sum, currencyCode, locale) : '',
+            }
+        }))
+
+        return acc
+    }, [])
+
+    totalVAT = totalSum * vat / 100
+    totalVAT.toFixed(2)
+
+    const documentData = {
+        header: {
+            generalDate: printDate.format('DD.MM.YYYY'),
+        },
+        company: {
+            name: get(organization, 'name') || '_______________________',
+            psrn: psrn || '_______________________',
+            tin: get(organization, 'tin') || '_______________________', 
+            iec: iec || '_______________________',
+            address: organizationAddress || '_______________________',
+            phone: '_______________________', 
+
+        },
+        bankDetails: {
+            accountNumber: get(invoices, '0.accountNumber') || '_______________________',
```

Comment:
lodash.get has a default value as third argument

accountnumber: get(invoices, '0.accountNumber', BLANK)

Reply:
I wouldn't use `lodash.get` at all :)

There is `optional chaining`, it are very good at coding for frontend with typescript.
The downside of `lodash.get` is that it ignores any type errors that typescript would highlight